cluster/gce/config-default.sh

```diff
@@ -254,6 +254,13 @@ ENABLE_DEFAULT_STORAGE_CLASS="${ENABLE_DEFAULT_STORAGE_CLASS:-true}"
 # Optional: Enable legacy ABAC policy that makes all service accounts superusers.
 ENABLE_LEGACY_ABAC="${ENABLE_LEGACY_ABAC:-true}" # true, false
 
+# Optional: use a default, fairly open PodSecurityPolicy
+ENABLE_POD_SECURITY_POLICY="${ENABLE_POD_SECURITY_POLICY:-true}"
+
+if [[ "${ENABLE_POD_SECURITY_POLICY:-}" == "true" ]]; then
+  ADMISSION_CONTROL="${ADMISSION_CONTROL},PodSecurityPolicy"
```

Comment:
That sounds like it's just generally the case. I've added an explicit comment to keep ResourceQuota at the end.